lib/live_view_demo/documentation.ex

```diff
@@ -50,11 +50,25 @@ defmodule LiveViewDemo.Documentation do
 
   @impl true
   def handle_call({:get, key}, _from, docs) do
+    # TODO fallback to function with less arity than required to cover invocations
+    # when default params are omitted or first param is omitted because of piped operations
```

Comment:
Not related with other changes in this PR. Just wanted to leave a note here to remember this limitation.